### DISCUSSION TITLE
Square Golf Monitor support

### DISCUSSION THREAD

**Discussion by GhostRider076** [springbok/MLM2PRO-GSPro-Connector#131] [Ideas]
Are you working on support of the Square Golf Monitor?

**Comment by wonder99** [comment]
it should already be there, though I don't have a way to test it.  A user has already requested it.

**Comment by springbok** [comment]
I think they have just released official support for gspro.

**Reply by GhostRider076** [reply]
Thanks.  I just looked and it is there.
